data/Pandora.Api/V1/Controllers/Terraform-Models.cs

```diff
@@ -255,6 +255,9 @@ private class ResourceIdMappingDefinition
 
         [JsonPropertyName("segmentName")]
         public string SegmentName { get; set; }
+
+        [JsonPropertyName("parent")]
```

Comment:
we'll want to change the json field name here too?

```suggestion
        [JsonPropertyName("parsedFromParentId")]
```